docs/versionpicker.html

```diff
@@ -0,0 +1,115 @@
+<!-- Start interactive version picker -->
+<div id="installation" v-cloak>
+  <header class="row">
+    <h3>Paste this into the <code>&lt;head&gt;</code> of your HTML:</h3>
+    <button type="button" @click="copyToClipboard">
+      <transition>
+        <span v-if="copyStatus === 'success'" class="emoji">✔</span>
+        <span v-else-if="copyStatus === 'failed'" class="emoji">❌</span>
+      </transition>
+      Copy to clipboard
+    </button>
+  </header>
+
+  <transition mode="out-in">
+    <pre :key="selectedVersion.fileSnippet"><code>{{selectedVersion.fileSnippet}}</code></pre>
+  </transition>
+
+  <h3>Version options:</h3>
+  <form action="./" method="post" @submit.prevent>
+    <label>
+      <input type="radio" value="dark" checked name="theme" v-model="versionOptions.theme" />
+      Dark theme&nbsp;<span class="emoji">🌙</span>
+      <span
+        v-if="!versionOptions.isStandalone"
+        class="emoji translucent"
+        title="Selected theme may be overwritten by theme settings on user's device"
+      >
+        / ☀
+      </span>
+    </label>
+    <label>
+      <input type="radio" value="light" name="theme" v-model="versionOptions.theme" />
+      Light theme&nbsp;<span class="emoji">☀</span>
+      <span
+        v-if="!versionOptions.isStandalone"
+        class="emoji translucent"
+        title="Selected theme may be overwritten by theme settings on user's device"
+      >
+        / 🌙
+      </span>
+    </label>
+    <br /><br />
+    <label>
+      <input type="checkbox" checked v-model="versionOptions.isStandalone" />
+      Enforce theme? (ignore <code>prefers-color-scheme</code>)
+    </label>
+    <br /><br />
+    <label>
+      <input type="checkbox" v-model="versionOptions.isLegacy" />
+      Support Internet Explorer and other legacy browsers?
+    </label>
+  </form>
+
+  <h3>Version info:</h3>
```

Comment:
This could be done with a [`<caption>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption) for the table below.

Reply:
Neat!

Do you want me to style the `<caption>` so it looks like the other `<h3>` elements?
Otherwise it looks like this:

&nbsp;  

![image](https://user-images.githubusercontent.com/30421456/59087635-8b837200-8905-11e9-838c-0eef036b2554.png)

Reply:
Done.